pupil_src/shared_modules/background_helper.py

```diff
@@ -56,6 +58,14 @@ def _wrapper(self, pipe, _should_terminate_flag, generator, *args, **kwargs):
             pipe.close()
             logger.debug('Exiting _wrapper')
 
+    def _fix_background_logger(self):
+        del logger.root.handlers[0]
+        handler = logging.StreamHandler()
+        handler.setFormatter(logging.Formatter(
```

Comment:
so this streams log messages to STDOUT, would it not be better to send the log messages through  the zmq IPC just like all other parts of pupil do? Or am i missing something? 

Alternative would be to send all log messages through the wrapper pipe... and then log them in the mainthread (which will also put them on the IPC.)

Reply:
I think you are right, @papr suggested this as well yesterday.

Reply:
Yes, there is a nice solution which I will push into the PR later today.